book/website/communication/open/embedding-ds-ai-disciplines.md

```diff
@@ -0,0 +1,161 @@
+(cm-embedding-ds-ai-disciplines)=
+# Embedding Data Science & AI across Disciplines: Context and Suggestions for Educators
+
```

Comment:
Add a note section here - you can use markdown notation to set up an "info" section